integration/spark2/src/main/scala/org/apache/spark/sql/parser/CarbonSparkSqlParser.scala

```diff
@@ -190,7 +191,7 @@ class CarbonHelperSqlAstBuilder(conf: SQLConf, parser: CarbonSpark2SqlParser)
     }
 
     val tableProperties = mutable.Map[String, String]()
-    properties.foreach{property => tableProperties.put(property._1, property._2)}
+    validatedProperties.foreach{property => tableProperties.put(property._1, property._2)}
```

Comment:
i think we can remove this code. After this we will extract the values so no point in creating mutable.Map. Can you please confirm this?

Reply:
Removing the mutable Map will lead to changes in the parameters of other methods. Should I work on that?

Reply:
Can't remove the Mutable Map as it is being updated in prepareTableModel method